backend/dl/detection.py

```diff
@@ -87,11 +91,20 @@ def write_to_csv(label_set):
             writer.writerows(label_set)
 
 
-def detection_img_drive(IMAGE_UPLOAD_FOLDER, detection_type, problem_type, transforms):
-    for x in os.listdir(IMAGE_UPLOAD_FOLDER):
-        if x != ".gitkeep":
-            img_file = os.path.join(os.path.abspath(IMAGE_UPLOAD_FOLDER), x)
-            break
+def detection_img_drive(trainspace_data: TrainspaceData):
+    uid = trainspace_data.uid
+    filename = trainspace_data.dataset_data["name"]
+    read_from_bucket(
```

Comment:
highly recommended to reach out to Austin to ask him how his trainspace data would look like for object detection? @farisdurrani